src/mit_core/ext.py

```diff
@@ -232,20 +232,22 @@ class LibcLib(AbstractInstruction):
     mit_lib['GET_{}'.format(register.name.upper())] = (
         len(mit_lib), None, None, '''\
     mit_state *inner_state;
-    POP_STACK_TYPE(S, mit_state *, &inner_state);
```

Comment:
We already have Python code to do these big loads and stores. It's in `instruction_gen.StackItem.load()`. Python also knows the size of `mit_state *`. Looks like we can factor out the commonality and do both in Python.

Reply:
Done.